dbt/adapters/spark/__init__.py

```diff
@@ -5,7 +5,7 @@
 from dbt.adapters.spark.impl import SparkAdapter
 
 from dbt.adapters.base import AdapterPlugin
-from dbt.include import spark
+from dbt.include import spark  # type: ignore
```

Comment:
Not sure why mypy started to complain things out of no where, just added ignore for now as it has been like this for a long time